### PR TITLE
Update Azure db connection string in configurations example

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ logging:
   output: "-"
 
 database:
-  connection_string: "postgres://user:pass@lakefs.rds.amazonaws.com:5432/postgres"
+  connection_string: "postgres://azureuser@lakefs-db:pass@lakefs-db.postgres.database.azure.com/postgres"
```
Comment (talSofer) [inline docs/reference/configuration.md:265]:
```suggestion
  connection_string: "postgres://user@lakefs-db:pass@lakefs-db.postgres.database.azure.com/postgres"
```

File: docs/reference/configuration.md [hunk 2]
```diff
@@ -262,7 +262,7 @@ logging:
   output: "-"
 
 database:
-  connection_string: "postgres://user:pass@lakefs.rds.amazonaws.com:5432/postgres"
+  connection_string: "postgres://user@lakefs-db:pass@lakefs-db.postgres.database.azure.com/postgres"
```
Comment (nopcoder) [inline docs/reference/configuration.md:265]: I don't think this syntax is supported. https://www.postgresql.org/docs/current/libpq-connect.html
check 34.1.1.2. Connection URIs

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: docs/reference/configuration.md (hunk 2 above)
Comment (talSofer) [reply docs/reference/configuration.md:265]: I have just setup an instance using this syntax :) that's why I'm suggesting this change

Comment (talSofer) [reply docs/reference/configuration.md:265]: Maybe there is a difference between Postgres versions?

**Review by johnnyaug — APPROVED** [review]

File: docs/reference/configuration.md (hunk 2 above)
Comment (talSofer) [reply docs/reference/configuration.md:265]: @nopcoder WDYT?

Comment (nopcoder) [reply docs/reference/configuration.md:265]: Don't think we need to update the documentation - the update example uses a user name called 'user@lakefs-db' which is a specific example.

**Comment by talSofer** [comment]
Cocnluded to discard this pr because username is 'user@lakefs-db' in this case
see https://github.com/treeverse/lakeFS/pull/3746#discussion_r932083766

**Pull request closed without merging**